packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.constants.js

```diff
@@ -28,9 +28,12 @@ export const TRACKING_NEW_PRODUCT_BANNER = {
     'dedicated::dedicatedCloud::details::datacenter::link-to-vmware-products',
 };
 
+export const OVERRIDE_IN = { IN: `en-IN` };
```

Comment:
In future we might need to add override for other countries as well. So dont use country specific variables.
```suggestion
export const LANGUAGE_OVERRIDE = { IN: `en-IN` };
```

Reply:
Done Anoop